pkg/reconciler/configuration/resources/revision_test.go

```diff
@@ -175,6 +175,54 @@ func TestMakeRevisions(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "with creator annotation from config",
+		configuration: &v1alpha1.Configuration{
+			ObjectMeta: metav1.ObjectMeta{
+				Namespace: "anno",
+				Name:      "config",
+				Annotations: map[string]string{
+					"serving.knative.dev/creator":      "admin",
+					"serving.knative.dev/lastModifier": "someone",
+				},
+				Generation: 10,
+			},
+			Spec: v1alpha1.ConfigurationSpec{
+				DeprecatedRevisionTemplate: &v1alpha1.RevisionTemplateSpec{
+					Spec: v1alpha1.RevisionSpec{
+						DeprecatedContainer: &corev1.Container{
+							Image: "busybox",
+						},
+					},
+				},
+			},
+		},
+		want: &v1alpha1.Revision{
+			ObjectMeta: metav1.ObjectMeta{
+				Namespace:    "anno",
+				GenerateName: "config-",
+				Annotations: map[string]string{
+					"serving.knative.dev/creator": "admin",
```

Comment:
This should be someone as they were the last person to edit this Configuration

```suggestion
					"serving.knative.dev/creator": "someone",
```